.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]: "
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug:**
+A clear and concise description of what the bug is.
+
+**To Reproduce:**
+Steps to reproduce the behavior:
+1. Step 1 '...'
+2. Step 2 '....'
+3. Step 3 '....'
+4. See error
+
+**Expected behavior:**
+A clear and concise description of what you expected to happen.
+
+**Reproducibility:**
+ - [ ] Always
+ - [ ] Sometimes
+ - [ ] Rarely
+
+**Screenshots:**
+If applicable, add screenshots to help explain your problem.
+
+**Environment:**
+ - OS: [e.g. iOS]
```

Comment:
We don't officially support iOS (and Apple is no longer using Intel chips) so this iOS example is very odd. Can you change it to Ubuntu?